Makefile

```diff
@@ -592,8 +590,6 @@ ifeq ($(PACKAGE_IN_CONTAINER), true)
 
 .PHONY: packages
 packages: dist packaging/fpm/$(UPTODATE)
-	@mkdir -p $(shell pwd)/.pkg
-	@mkdir -p $(shell pwd)/.cache
```

Comment:
Isn't this still required if you run `make BUILD_IN_CONTAINER=false packages`?

Reply:
This _seems_ to work fine for me - what needs these directories?

Reply:
These are used by the go build system, but I actually don't think we need to create them (they will be created by go if required).